flake.nix

```diff
@@ -171,8 +180,30 @@
               source ${writeScript ./nix/shell.sh}
             '';
           });
-
-        # Used to backup the devShell to s3 for caching.
+        devShells.awslcfips2022 = devShells.default.overrideAttrs
+          (finalAttrs: previousAttrs: {
+            # Re-include cmake to update the environment with a new libcrypto.
+            buildInputs = [ pkgs.cmake aws-lc-fips-2022 ];
+            S2N_LIBCRYPTO = "awslc-fips-2022";
+            shellHook = ''
+              echo Setting up $S2N_LIBCRYPTO environment from flake.nix...
+              export PATH=${openssl_1_1_1}/bin:$PATH
```

Comment:
Why are these pathing to openssl_1_1_1?

Reply:
The integration tests `s_client` needs 1.1.1 https://github.com/aws/s2n-tls/issues/3963 .
@jmayclin was talking about addressing this, but I've lost track.

Reply:
Ahh got it ok